tests/queries/models.py

```diff
@@ -264,6 +264,14 @@ class Related(models.Model):
     custom = models.ForeignKey(CustomPk, models.CASCADE)
 
 
+class RelatedNull(models.Model):
+    custom = models.ForeignKey(CustomPk, models.CASCADE, null=True)
```

Comment:
It usually isn't a problem. There always the option to add another field to that model if making the existing field nullable causes an issue.